Funky-Chickens/workFORCEmanagement
# Got all the employees, and one employee, and pugged em both to the DOM

To test:

Go to localhost:4000/employees
Check to see all employees

AND 

go to localhost:400/employees/[id of your choice between 1-10]
Check to make sure there is data for one employee
